services/core/cards-publication/src/main/modeling/swagger.yaml

```diff
@@ -731,6 +731,23 @@ paths:
           description: created
           schema:
             $ref: '#/definitions/CardCreationReport'
+    delete:
+      tags:
+        - cards
+        - deletion
+      summary: delete cards
+      description: Delete cards before specified date from OperatorFabric
```

Comment:
Add explanation about the behaviour : use endDate field and startDate when ... etc..

Reply:
Done